plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/discoverer/XcpServerDiscoverer.java

```diff
@@ -115,7 +115,7 @@
     protected String _guestNic;
     protected boolean _setupMultipath;
     protected String _instance;
-    private String xs620snapshothotfix = "Xenserver-Vdi-Copy-HotFix";
+    // private String xs620snapshothotfix = "Xenserver-Vdi-Copy-HotFix";
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable xs620snapshothotfix was not in use. Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.